Dockerfile

```diff
@@ -9,8 +9,8 @@ ENV LANGUAGE en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
 
 RUN dnf --setopt install_weak_deps=false -y install \
-		appstream gcc gettext git glib2-devel gnome-shell \
-		gnome-desktop-testing lcov meson npm \
+		appstream desktop-file-utils gcc gettext git glib2-devel \
+		gnome-shell gnome-desktop-testing lcov meson npm \
 		xorg-x11-server-Xvfb zip && \
 	dnf clean all && \
```

Comment:
It's amazing how this tiny list of installs blows up into _such_ a Huge. Package. Transaction! (Though, I suppose nothing containing `gnome-shell` can ever be considered "tiny".)

Still... what's `lcov` doing in there, anyway? I just checked, and there's not a single package in the entire Fedora collection that either `Requires` or `Recommends` it (which is actually surprising!), and I can't see how or where it's actually being used for anything, being as it's a reporting tool for test coverage data produced by C/C++ compilers.

(...Really not critical, anyway — `lcov` itself _actually is_ tiny, and it's lost in the noise of the behemoths surrounding it. Tho... then again, it probably is to blame for dragging in a bunch of the Perl packages that end up in the install set — not that Git needs any help on that front.)

Reply:
> Still... what's lcov doing in there, anyway?

Probably doesn't need to be. I had planned to add a coverage report at the time, but the GJS Promise queue implementation was breaking the coverage generation. I gave it another shot about a week ago but still had some problems getting it to run to completion. It's probably pretty depressing anyways :)

Reply:
I'll merge this one and open a new one to remove that package for now. If someone else wants to take a crack at it they can :)